.install/install_llvm.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+OS_TYPE=$(uname -s)
+
+MODE=$1 
+VERSION=$2
+
+
+
+if [[ $OS_TYPE == Darwin ]]; then
+    brew install llvm@$VERSION
+else
+    $MODE apt install lsb-release wget software-properties-common gnupg
```

Comment:
If we use `apt` is it working on all Linux based OSs?

Reply:
yes, let's start here, with the common OSs to devs,  and see if we need to add support to other OSs as well